rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2629,7 +2636,7 @@ namespace rsx
 		}
 
 		zcull_ctrl->set_enabled(this, zcull_rendering_enabled);
-		zcull_ctrl->set_active(this, zcull_rendering_enabled && testing_enabled && zcull_surface_active);
+		zcull_ctrl->set_active(this, zcull_surface_active);
```

Comment:
Make zcull_surface_active a local variable as it no longer represents a global status. It is possible to have a zcull surface active even with tests disabled.